src/Tab/TabCollectionProvider.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\Module\Mbo\Tab;
+
+use Doctrine\Common\Cache\CacheProvider;
+use PrestaShop\Module\Mbo\ExternalContentProvider\ExternalContentProvider;
+use Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException;
+
+class TabCollectionProvider implements TabCollectionProviderInterface
+{
+    const CACHE_KEY = 'recommendedModules';
+
+    const CACHE_LIFETIME_SECONDS = 604800; // 7 days same as defined in Core
```

Comment:
Why not reuse the constant defined in the Core ?

Reply:
What is constant defined in Core ?

Reply:
`7 days same as defined in Core` ?

Reply:
Its not defined as a constant in core

Reply:
See https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L2213